app/data/models/stage-configuration.js

```diff
@@ -0,0 +1,31 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const stageConfiguration = sequelize.define('stage_configuration', {
+    id: {
+      type: DataTypes.SMALLINT,
+      primaryKey: true,
+      allowNull: false
+    },
+    stage: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    role: {
+      type: DataTypes.JSONB,
+      allowNull: false
+    },
+    stepNumber: {
+      type: DataTypes.SMALLINT,
+      allowNull: false
+    },
+    action: {
+      type: DataTypes.JSONB,
+      allowNull: false
+    }
+  },
+  {
+    freezeTableName: true,
+    tableName: 'stage_configuration'
```

Comment:
Can you add the following in here `timestamps: false`, I can't remember the exact issue I was having during the spike but this resolved it